recipe/thirdparty/utils.go

```diff
@@ -85,6 +86,10 @@ func parseUser(value interface{}) (*tpmodels.User, error) {
 	if err != nil {
 		return nil, err
 	}
+	if strings.Contains(user.ThirdParty.UserID, "|") {
```

Comment:
are you sure all recipe impl functions call this function here?

Reply:
yes, I have verified this